include/clasp/core/externalObject.h

```diff
@@ -1,38 +1,78 @@
-/*
-    File: externalObject.h
-*/
-
-/*
-Copyright (c) 2014, Christian E. Schafmeister
- 
-CLASP is free software; you can redistribute it and/or
-modify it under the terms of the GNU Library General Public
-License as published by the Free Software Foundation; either
-version 2 of the License, or (at your option) any later version.
- 
-See directory 'clasp/licenses' for full details.
- 
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
-*/
-/* -^- */
-#ifndef ExternalObject_H
-#define ExternalObject_H
-
-#include <clasp/core/foundation.h>
-#include <clasp/core/object.h>
-#include <clasp/core/standardObject.h>
+// -*- Mode: C++; tab-width: 2; indent-tabs-mode: c++; c-basic-offset: 2 -*-
+//
+//   ██████╗    ██╗          █████╗     ███████╗    ██████╗
+//  ██╔════╝    ██║         ██╔══██╗    ██╔════╝    ██╔══██╗
+//  ██║         ██║         ███████║    ███████╗    ██████╔╝
+//  ██║         ██║         ██╔══██║    ╚════██║    ██╔═══╝
+//  ╚██████╗    ███████╗    ██║  ██║    ███████║    ██║
+//   ╚═════╝    ╚══════╝    ╚═╝  ╚═╝    ╚══════╝    ╚═╝
+//
+// ===========================================================================
+//    C L A S P - COMMON LISP AND C++
+// ===========================================================================
+//
+// Copyright (c) 2014, Christian E. Schafmeister
+//
+// CLASP is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Library General Public
+// License as published by the Free Software Foundation; either
+// version 2 of the License, or (at your option) any later version.
+//
+// See directory 'clasp/licenses' for full details.
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+// ===========================================================================
+
+//! @file   externalObject.h
+//! @author Christian E. Schafmeister
+//! @brief
```

Comment:
What do you *need* an ASCII graphic header for?

Either we're going to have a consistent source style throughout the repository, or I'm going to resign from all responsibilities I've taken up for this project.